### DISCUSSION TITLE
Creating Folder and File at the same time - Not working as we expected

### DISCUSSION THREAD

**Discussion by bhuppal** [zed-industries/zed#9533] [Feature Requests]
In VSCode, I can able to do as follow:

- In the File Explorer, Right click-> Create New Folder -> type as "MyFolder/readme.txt" - Result is it creates Folder call "MyFolder" under that a file "readme.txt"

But same thing when I tried with Zed, it created 2 folders as "MyFolder" and "readme.txt" another folder. Just want to bring to our attention. 

Thanks,
Bhuppal

**Comment by bhuppal** [comment]
![image](https://github.com/zed-industries/zed/assets/4120729/6fb27a2e-1c76-4241-9555-cdf09fdf84a7)

**Comment by failable** [comment]
I think that's because the menu item is "New Folder" instead of "New File"...the later just works fine. I am afraid that's desired behavior.
